### PR TITLE
Added disable prop to Certify

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#7214

*Description of changes:*
* Added prop to disable editing of the field when required.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/Certify/Certify.stories.ts [hunk 1]
```diff
@@ -21,5 +21,6 @@ Default.args = {
   isCertified: false,
   message: 'Note: It is an offence to make a false or misleading statement in respect of a material fact in a \n' +
   'record submitted to the Corporate Registry for filing. See section 427 of the Business Corporations Act.',
-  entityDisplay: 'BC Company'
+  entityDisplay: 'BC Company',
+  disableEdit: false
```
Comment (severinbeauvais) [inline src/components/Certify/Certify.stories.ts:25]: Is it possible to add a small unit test or 2 to validate this new prop?

**Review by severinbeauvais — APPROVED** [review]
LGTM but a test would be nice :)

File: src/components/Certify/Certify.stories.ts (hunk 1 above)
Comment (cameron-freshworks) [reply src/components/Certify/Certify.stories.ts:25]: 100%! Will do 👍

Comment (cameron-freshworks) [reply src/components/Certify/Certify.stories.ts:25]: Updated!

File: tests/unit/Certify.spec.ts [hunk 2]
```diff
@@ -230,4 +239,11 @@ describe('Certify', () => {
     expect(wrapper.find('#business-email').text()).toContain('mockBusinessEmail')
     expect(wrapper.find('#completing-party-email').text()).toContain('mockPartyEmail')
   })
+
+  it('disables input field when prop is passed true', () => {
+    const wrapper: Wrapper<Certify> = createComponent(null, true, false, null, false, false, [], true, 'mockBusinessEmail', 'mockPartyEmail', true)
+
+    expect(wrapper.find('#certified-by-textfield').exists()).toBe(true)
+    expect(wrapper.find('#certified-by-textfield').attributes('disabled')).toBeTruthy()
```
Comment (severinbeauvais) [inline tests/unit/Certify.spec.ts:247]: Great!

Comment (cameron-freshworks) [reply tests/unit/Certify.spec.ts:247]: 😁  Thanks for the reminder on the tests

**Pull request merged**